src/Classes/iTones/iTones_Playlist.php

```diff
@@ -513,6 +548,78 @@ public static function getPlaylistFromWeights($playlists_to_ignore = [])
         return self::getInstance(CoreUtils::biasedRandom($result));
     }
 
+    /**
+     * Uses weighted playout values to select a random Playlist from a category, returning it.
+     *
+     * Only includes Playlists with a currently running slot, and a Track.
+     *
+     * @param int $categoryId
```

Comment:
Here you have an int defined, to the API may enforce this, but still, would like to validate properly 👍